vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -145,13 +145,21 @@ bool TryUpdateBackgroundBrush(
   return false;
 }
 
-template <class T>
-void UpdateCornerRadius(
```

Comment:
I think the more efficient way is change the signature of this function:
```
void UpdateConrnerRadius(ShadowNodeBase *node, const T&element, ShadowCorners corner, PropertValue const& value) {
  if (value.isNumber()) {
    // set corner radius
  }
  else {
   // Clear
  }
}
```

Reply:
Ah, true! Well, less code and more maintainable. Hopefully, the compiler gives a similar result either way. Done.